src/tools/ethLoader/ethLoaderLib/EthMaintainer.cpp

```diff
@@ -815,8 +866,8 @@ std::string EthMaintainer::processMoreInfoReplies(EthBoardList &boardlist)
 
     while(mSocket.ReceiveFrom(rxipv4addr, rxipv4port, mRxBuffer, sizeof(mRxBuffer), 500) > 0)
     {
-        eOuprot_cmd_MOREINFO_REPLY_t *moreinfo = (eOuprot_cmd_MOREINFO_REPLY_t*) mRxBuffer;
```

Comment:
Just a memo to myself for later: so in here we don't need / manage a `eOuprot_cmd_MOREINFO2_REPLY_t`, or do we?.